functions/lib/integration/import-product.js

```diff
@@ -120,7 +120,7 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
           const tiny = new Tiny(tinyToken)
 
           const handleTinyStock = ({ produto, tipo }, tinyProduct) => {
-            let quantity = Number(produto.saldo) || Number(produto.estoqueAtual)
+            let quantity = (Number(produto.saldo) || Number(produto.estoqueAtual)) - Number(produto.saldoReservado)
```

Comment:
```suggestion
            let quantity = Number(produto.saldo) || Number(produto.estoqueAtual)
            if (produto.saldoReservado) {
              quantity -= Number(produto.saldoReservado)
            }
```

evitar erro com NaN se `produto.saldoReservado` indefinido...